test/data/stages/gunzip/b.mpp.json

```diff
@@ -0,0 +1,66 @@
+{
+  "version": "2",
+  "sources": {
+    "org.osbuild.inline": {
+      "items": {
+        "sha256:f950375066d74787f31cbd8f9f91c71819357cad243fb9d4a0d9ef4fa76709e0": {
+          "encoding": "base64",
+          "data": "SGFsbG8gV2VsdAo="
+        }
+      }
+    }
+  },
+  "pipelines": [
+    {
+      "mpp-import-pipeline": {
+        "path": "../manifests/f34-build-v2.json",
+        "id": "build"
+      },
+      "runner": "org.osbuild.fedora34"
+    },
+    {
+      "name": "tree-compressed",
+      "build": "name:build",
+      "stages": [
+        {
+          "type": "org.osbuild.gzip",
+          "inputs": {
+            "file": {
+              "type": "org.osbuild.files",
+              "origin": "org.osbuild.source",
+              "references": {
+                "sha256:f950375066d74787f31cbd8f9f91c71819357cad243fb9d4a0d9ef4fa76709e0": {}
+              }
+            }
+          },
+          "options": {
+            "filename": "compressed.gz"
+          }
+        }
+      ]
+    },
+    {
+      "name": "tree",
+      "build": "name:build",
+      "stages": [
+        {
+          "type": "org.osbuild.gzip",
```

Comment:
Shouldn't this be org.osbuilt.gunzip?

Reply:
gahhh, of course.

Reply:
Fixed and actually verified the contents of the tree this time. Sorry about that. Long day.
```
/w/osbuild → ./osbuild-cli --libdir . --store /tmp/store/ --output-directory /tmp/osbuild test/data/stages/gunzip/b.json --checkpoint build --export tree                                                        [gunzip-stage | M2 A40]
Pipeline build: cd93456be1ecb4a3e63f2aba6058fb888409b4d3dd8f6c7aa9db5c1232b0719c
Pipeline tree-compressed: 9b097ea98423154bc553595e3e290e9371008c39552adde1184b73156d0d7d1c
org.osbuild.gzip: 9b097ea98423154bc553595e3e290e9371008c39552adde1184b73156d0d7d1c {
  "filename": "compressed.gz"
}
/usr/lib/tmpfiles.d/journal-nocow.conf:26: Failed to resolve specifier: uninitialized /etc detected, skipping
All rules containing unresolvable specifiers will be skipped.

⏱  Duration: 0s
Pipeline tree: 1f01341931a91b1f42865ed5867ac06c9ea31f1d72ac0f4fe72d21007d979e21
org.osbuild.gunzip: 1f01341931a91b1f42865ed5867ac06c9ea31f1d72ac0f4fe72d21007d979e21 {
  "path": "uncompressed.txt"
}
/usr/lib/tmpfiles.d/journal-nocow.conf:26: Failed to resolve specifier: uninitialized /etc detected, skipping
All rules containing unresolvable specifiers will be skipped.
/usr/bin/sh: warning: setlocale: LC_ALL: cannot change locale (en_GB.UTF-8)

⏱  Duration: 0s
build:    	cd93456be1ecb4a3e63f2aba6058fb888409b4d3dd8f6c7aa9db5c1232b0719c
tree-compressed:	9b097ea98423154bc553595e3e290e9371008c39552adde1184b73156d0d7d1c
tree:     	1f01341931a91b1f42865ed5867ac06c9ea31f1d72ac0f4fe72d21007d979e21
root@cfcaf148dd3a ▢ /w/osbuild → cat /tmp/osbuild/tree/uncompressed.txt                                                                                                                                                              
───────┬─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
       │ File: /tmp/osbuild/tree/uncompressed.txt
───────┼─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
   1   │ Hallo Welt

Reply:
Should be fixed!

Reply:
looks better